File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/swift/F08.P1.swift
func probe() -> Int32 {
// BEGIN PROBE F08.P1
    let m = Int32.max
    let s = m.addingReportingOverflow(1)
    let o = s.overflow ? 0 : s.partialValue
    return o
// END PROBE F08.P1
}

print(probe())

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/cpp/F08.P1.cpp
#include <climits>
#include <cstdio>

int overflow_at_max() {
// BEGIN PROBE F08.P1
    int m = INT_MAX;
    int o = m + 1;
    return o;
// END PROBE F08.P1
}

int main() { std::printf("%d\n", overflow_at_max()); }
